LiteDB/Engine/Disk/DiskWriterQueue.cs

```diff
@@ -76,15 +73,15 @@ public void Wait()
         /// <summary>
         /// Execute all items in queue sync
         /// </summary>
-        private void ExecuteQueue()
+        private async Task ExecuteQueue()
         {
             try
             {
                 while (true)
                 {
                     if (_queue.TryDequeue(out var page))
                     {
-                        WritePageToStream(page);
+                        await WritePageToStream(page).ConfigureAwait(false);
```

Comment:
Please, remove this and other `ConfigureAwait`s, they aren't doing not here, just polluting the code

Reply:
Removed